Godeps/Godeps.json

```diff
@@ -125,6 +120,11 @@
 			"Rev": "c8388a52ec7a328b6aee3c48d31bd7f3e0b14aeb"
 		},
 		{
+			"ImportPath": "github.com/docker/docker/api/types/volume",
```

Comment:
@kevinetc123 let's keep this PR limited to the described changes, and not add other dependencies.